### PR TITLE
Fixed case when file is not present on cloud

### PR DESCRIPTION
**Pull request by vimutter**

### CODE REVIEW

File: lib/paperclip/storage/fog.rb [hunk 1]
```diff
@@ -170,6 +170,7 @@ def copy_to_local_file(style, local_dest_path)
         log("copying #{path(style)} to local file #{local_dest_path}")
         ::File.open(local_dest_path, 'wb') do |local_file|
           file = directory.files.get(path(style))
+          return false unless file
           local_file.write(file.body)
```
Comment (tute) [inline lib/paperclip/storage/fog.rb]: I'd write this as:
```ruby
if file = directory.files.get(path(style))
  local_file.write file.body
end
```
Do you think it reads better?

**Comment by tute** [comment]
This looks good to me. What was the behaviour before your fix? What is it now? Can you add that information to your commit message? Then we'll merge. Thanks!

File: lib/paperclip/storage/fog.rb (hunk 1 above)
Comment (vimutter) [inline lib/paperclip/storage/fog.rb]: @tute Usually we avoid assignment in conditionals as it always triggers internal question whether this line is typo or not. I.e. it requires check of surrounding of the line, which is sometimes ok. 
I.e. I can change this line if you wish so.

**Comment by vimutter** [comment]
@tute Currently it fails with NoMethodError on nil when file is not there. And since `nil` is proper response from fog on missing files and this method already returns false if any error happen (see line 178) I decided that returning false here would be properly handled on client code.

File: lib/paperclip/storage/fog.rb (hunk 1 above)
Comment (tute) [inline lib/paperclip/storage/fog.rb]: > @tute Usually we avoid assignment in conditionals as it always triggers internal question whether this line is typo or not. I.e. it requires check of surrounding of the line, which is sometimes ok. 
> I.e. I can change this line if you wish so.

I don't feel very strongly either way, so following the Hound (who didn't bark at this line) is good with me. 👍

**Comment by tute** [comment]
> Currently it fails with NoMethodError on nil when file is not there. And since nil is proper response from fog on missing files and this method already returns false if any error happen (see line 178) I decided that returning false here would be properly handled on client code.

Great! Can you please squash your commit together, rebase against latest master, and add this context to the commit message? Thanks!

**Comment by tute** [comment]
> Can you please squash your commit together, rebase against latest master, and add this context to the commit message? Thanks!

Did it myself, and pushed to master. Thank you! 🎉

**Pull request closed without merging**